assignment4/src/app/odd/odd.component.html

```diff
@@ -0,0 +1,2 @@
+<h1 style="position: relative;">{{oddEvent}}</h1>
```

Comment:
Adding inline css is a **very** bad coding practice.

Angular is giving you a whole `.css` file for a reason! Use it ;)